crates/matrix-sdk-base/src/latest_event.rs

```diff
@@ -408,10 +406,30 @@ mod tests {
 
         assert_matches!(
             is_suitable_for_latest_event(&event),
-            PossibleLatestEvent::NoUnsupportedMessageLikeType
+            PossibleLatestEvent::YesSticker(SyncStickerEvent::Original(_))
         );
     }
 
+    #[test]
+    fn test_different_types_of_messagelike_are_unsuitable() {
+        let event = AnySyncTimelineEvent::MessageLike(AnySyncMessageLikeEvent::UnstablePollResponse(
+            SyncUnstablePollResponseEvent::Original(OriginalSyncMessageLikeEvent {
+                content: UnstablePollResponseEventContent::new(
+                    vec![ String::from("option1") ],
+                    owned_event_id!("$1"),
+
+                ).into(),
+                event_id: owned_event_id!("$2"),
+                sender: owned_user_id!("@a:b.c"),
+                origin_server_ts: MilliSecondsSinceUnixEpoch(UInt::new(2123).unwrap()),
+                unsigned: MessageLikeUnsigned::new(),
+            })));
+
+        assert_matches!(
+            is_suitable_for_latest_event(&event),
+            PossibleLatestEvent::NoUnsupportedMessageLikeType
+        );
+    }
```

Comment:
ubernit, sorry :D (I'll apply and squash it with your PR

```suggestion
    }

```